### PR TITLE
don't expose the config disk option.

### PR DESCRIPTION
**Pull request by andyliuliming**

### CODE REVIEW

File: src/bosh_azure_cpi/lib/cloud/azure/cloud.rb [hunk 1]
```diff
@@ -333,7 +334,7 @@ def create_disk(size, cloud_properties, vm_cid = nil)
                 vmss_instance = @azure_client.get_vmss_instance(resource_group_name, instance_id_obj.vmss_name, instance_id_obj.vmss_instance_id)
                 location = vmss_instance[:location]
                 instance_type = vmss_instance[:sku][:name]
-                zone = vmss_instance[:zones][0]
+                zone = vmss_instance[:zones][0] unless vmss_instance[:zones].nil?
```
Comment (bingosummer) [inline src/bosh_azure_cpi/lib/cloud/azure/cloud.rb]: When the VMSS' zones are nil?

BTW, do we support [both single-zone and zone-redundant scale sets](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-use-availability-zones)?

Comment (andyliuliming) [reply src/bosh_azure_cpi/lib/cloud/azure/cloud.rb]: if we did not specify the zones for the vmss, then the zones would be nil.

Comment (andyliuliming) [reply src/bosh_azure_cpi/lib/cloud/azure/cloud.rb]: and for the zone-redundant.
customer will have option to specify the vmss zones in the cloud properties.

**Comment by andyliuliming** [comment]
@bingosummer yes, but I will only remove the spec config part. if do the full change, we will get them back when merging to develop branch.

**Pull request merged**